edx_sga/sga.py

```diff
@@ -215,6 +215,7 @@ def student_view(self, context=None):
         )
         fragment.add_css(_resource("static/css/edx_sga.css"))
         fragment.add_javascript(_resource("static/js/src/edx_sga.js"))
+        fragment.add_javascript(_resource("static/js/src/jquery.tablesorter.min.js"))
```

Comment:
I have a question for my own information.  I see you are including the minimized version of this library.  Doesn't edX already manage compressing javascript?  Or is it a best practice to use minimized versions of javascript libraries?